manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Quadren.Arc.Prerelease
+PackageVersion: PBv1.1.3
+PackageLocale: en-US
+Publisher: Quadren
+PackageName: Arc
+License: MIT License

```

```suggestion
License: Proprietary
```
I can't find the license https://github.com/quadren/arc/ or on the package website https://quadren.github.io/arc/. Can you provide a link to the package's license if it exists? If no solid information about the license is available, this field should be set to `Proprietary`.